src/pages/iou/request/step/IOURequestStepParticipants.tsx

```diff
@@ -127,6 +128,8 @@ function IOURequestStepParticipants({
     const [introSelected] = useOnyx(ONYXKEYS.NVP_INTRO_SELECTED, {canBeMissing: true});
     const personalPolicy = useMemo(() => Object.values(allPolicies ?? {}).find((policy) => policy?.type === CONST.POLICY.TYPE.PERSONAL), [allPolicies]);
 
+    const currentUserPersonalDetails = useCurrentUserPersonalDetails();
```

Comment:
In these cases should we just get the accountID if we only need that in the component? 
``` suggestion
    const {accountID} = useCurrentUserPersonalDetails();
```